extensions/data-plane-selector/data-plane-selector-api/src/main/java/org/eclipse/edc/connector/dataplane/selector/api/DataplaneSelectorApiController.java

```diff
@@ -22,40 +24,74 @@
 import jakarta.ws.rs.core.MediaType;
 import org.eclipse.edc.connector.dataplane.selector.spi.DataPlaneSelectorService;
 import org.eclipse.edc.connector.dataplane.selector.spi.instance.DataPlaneInstance;
+import org.eclipse.edc.spi.EdcException;
+import org.eclipse.edc.spi.result.Result;
+import org.eclipse.edc.transform.spi.TypeTransformerRegistry;
+import org.eclipse.edc.web.spi.exception.InvalidRequestException;
 
-import java.util.List;
+import java.util.function.Supplier;
 
+import static jakarta.json.stream.JsonCollectors.toJsonArray;
+import static java.util.Optional.ofNullable;
 import static org.eclipse.edc.web.spi.exception.ServiceResultHandler.exceptionMapper;
 
 @Consumes({ MediaType.APPLICATION_JSON })
 @Produces({ MediaType.APPLICATION_JSON })
-@Path("/instances")
```

Comment:
do you mean to register the new controller in two paths? because we not only changed the path, but also JSON-LD-ified it, which is likely the larger break...

Reply:
yes, because it will break all the samples and likely the MVD all of a sudden, I'll keep the old, deprecated one, also without tests, at least for another version just to give time to adapt

Reply:
can't we just update the other projects? Did a quick search, and I only found these occurrences:
- Samples: example `curl` commands in README.md
- MVD: a postman collection
both should be easy to fix.

In Tractus-X there was only occurrence, which is also easily fixed, because it wrongly uses the `DataPlaneService` directly, where it should use the `DataPlaneClient`.

Reply:
ok, fine for me, it was just to avoid complaints on every release